spec/README.md

```diff
@@ -107,12 +107,22 @@ defined in the default registry.
 > (such as due to the release of new CLDR versions).
 
 Updates to this specification will only reserve, define, or require
-function names or function option names
-consisting of characters in the ranges a-z, A-Z, and 0-9.
-All other names in these categories are reserved for the use of implementations or users.
+function identifiers and function option identifiers
+which satisfy either of the following two requirements:
+- Includes no namespace,
+  and has a name consisting of characters in the ranges a-z, A-Z, and 0-9.
```

Comment:
I don't think this is necessary?

Reply:
It gives us a bit more flexibility in the future, in case we want to use foo.bar in the future

I feel less strongly about '_'.

Reply:
For function options we're already using camelCase, but I don't oppose reserving `.` as well.

@aphillips, if you agree, merge this suggestion?
```suggestion
  and has a name consisting of characters in the ranges a-z, A-Z, and 0-9,
  and the character U+002E FULL STOP `.`.
```

Reply:
I think I'd rather merge the first suggestion. Note that reserving `.`, `-`, and `_` does not prevent implementations or users from using these characters (this is what namespaces are for). A certain amount of flexibility is good for "future us". TBH, I think `.` is my least favorite, since it smacks of properties. The symbols don't meaning anything, of course.

Reply:
I would rather allow us to use '.' in the future, for future proofing. It doesn't prevent others from using that value.

Reply:
@macchiati Do you have an objection to the other two characters? I have no objection to `.`